pylint_odoo/test/main.py

```diff
@@ -8,7 +8,7 @@
 from pylint_odoo import misc
 
 
-EXPECTED_ERRORS = 46
+EXPECTED_ERRORS = 50
```

Comment:
Suggestion - to avoid the annoying PR conflicts you could write something like this: 
```python
EXPECTED_ERRORS = 46
EXPECTED_ERRORS += 4
```